### PR TITLE
update circle conf

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by agramfort** [comment]
@GuillaumeFavelier @ja-che this is now all good 

https://56-259908992-gh.circle-artifacts.com/0/dev/auto_examples/plot_meg_data_example.html#sphx-glr-auto-examples-plot-meg-data-example-py

the magic line is:

export OMP_NUM_THREADS=1;

we don't need this joblib magic anymore ! 

feel free to merge this one.

not sure about the travis failure it seems unrelated

**Comment by GuillaumeFavelier** [comment]
Very nice 👌

**Comment by ja-che** [comment]
Thank you @agramfort and @GuillaumeFavelier.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,41 +1,69 @@
...
+    build_docs:
+      docker:
+        - image: circleci/python:3.8.5-buster
+      steps:
+        - checkout
+        - run:
+            name: Set BASH_ENV
+            command: |
+              set -e
+              echo "set -e" >> $BASH_ENV
```
Comment (ja-che) [inline .circleci/config.yml:19]: image

Do think those lines could create the Travis error ?

**Comment by agramfort** [comment]
No. You have the same errors in your other PRs

**Comment by ja-che** [comment]
This error occured when I added .circleci/config.yml, it was done in the last commit on the main branch.

Do you want to merge this PR and solve this travis error in "new example PRs" #28 or #26  ?

**Comment by agramfort** [comment]
Yes let’s fix Travis is a new PR

**Pull request merged**